### ISSUE TITLE
Follow Firefox light/dark theme

### ISSUE THREAD

**Issue by ValdikSS** [fastaddons/GroupSpeedDial#113]
**Is your feature request related to a problem? Please describe.**
Firefox 96 now has native light and dark theme. When you switch them, it also switches `prefers-color-scheme` CSS media feature, so all the websites know you have dark theme.
Right now Group Speed Dial can't detect Firefox theme and require the theme to be selected in the options.

**Describe the solution you'd like**
I'd like Group Speed Dial to automatically follow Firefox theme.

**Comment by ValdikSS** [comment]
Settings do follow system dark mode

![Image](https://github.com/user-attachments/assets/e4133096-c033-4826-bc85-4912bfe964f9)

But the group dial themes do not

![Image](https://github.com/user-attachments/assets/bfc8de14-5ca6-41c9-986b-9185d6ce8560)

**Comment by Juraj-Masiar** [comment]
Yeah, these two are two separate modules, also in time (theme color was here way before the UI dark mode).

Changing theme can have some unexpected consequences - specifically the thumbnails of the "group dials" can show the "wrong theme" preview if the thumbnail was created when the other theme was active.

But I guess not everyone uses those, so I could add a new option (below the "Controlled by system"), something like "Auto-switch also Theme style".

**Comment by ValdikSS** [comment]
Yes please, I don't mind white background of the dials preview much. The speed dial background is that's add white light at night, the dials itself are negligible.

**Comment by ValdikSS** [comment]
@Juraj-Masiar would appreciate the update!

**Comment by Juraj-Masiar** [comment]
Hello,
I'm sorry for the huge delay. I'm very busy all the time :(.
Many other features got added to my todo list before this issue.
I'm putting it on my "next release" todo list, so unless there is a problem implementing this, it should be added in the upcoming 28.4 release.
